user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Security_NATS.md

```diff
@@ -8,100 +8,128 @@ By default, NATS does **not** employ TLS encryption, leaving communication susce
       }
-      ```
+   }
+   ```
 
-      > [!NOTE]
-      > The `ca_file` property is only required when self-signed certificates are used.
+   > [!NOTE]
+   > The `ca_file` property is only required when self-signed certificates are used.
 
-   1. In the *Services* tab in the Windows Task Manager, stop the NATS service.
+1. In the *Services* tab in the Windows Task Manager, stop the NATS service.
 
-   1. In the *Services* tab in the Windows Task Manager, stop the NAS services.
+1. In the *Services* tab in the Windows Task Manager, stop the NAS services.
 
-   1. In the *Services* tab in the Windows Task Manager, start the NATS service.
+1. In the *Services* tab in the Windows Task Manager, start the NAS service.
 
-      This should automatically trigger the start of the NAS service as well.
+1. In the *Services* tab in the Windows Task Manager, start the NATS service.
 
-   1. Verify whether TLS encryption is enabled for inter-node communication:
+1. Verify whether TLS encryption is enabled for inter-node communication:
 
-      1. Go to `http://localhost:8222/varz` in a browser.
+   1. Go to `http://localhost:8222/varz` in a browser.
 
-      1. Check whether the `urls` property encompasses all the IP addresses of the NATS nodes configured with TLS.
+   1. Check whether the `urls` property encompasses all the IP addresses of the NATS nodes configured with TLS.
 
-      1. Check whether the `tls_required` and `tls_verify` properties are appropriately configured to **true**.
+   1. Check whether the `tls_required` and `tls_verify` properties are appropriately configured to **true**.
 
-      For example:
+   For example:
 
-      ```json
-        "cluster": {
-          "urls": [
-            "[NATS_NODE_IP_ADDRESS]:6222",
-          ],
-          "tls_required": true,
-          "tls_verify": true
-        }
-      ```
+   ```json
+      "cluster": {
+         "urls": [
+         "[NATS_NODE_IP_ADDRESS]:6222",
+         ],
+         "tls_required": true,
+         "tls_verify": true
+      }
+   ```
 
 ## Enabling DataMiner-to-NATS node TLS communication
 
+To enable DataMiner-to-NATS node TLS communication, first [install the TLS certificate](#install-the-tls-certificate), and then [configure the TLS certificate](#configure-the-tls-certificate).
+
 > [!IMPORTANT]
 > TLS encryption for communication between software and NATS nodes is available starting from DataMiner 10.4.3 [CU0]/10.5.0<!-- RN 38302 --> with support for CA-signed certificates (Certificate Authority-Signed Certificates). While support for self-signed certificates will be introduced later, keep in mind that for now only CA-signed certificates are supported and attempting to use self-signed certificates will result in DataMiner startup failures.
 
-1. Install the certificate in the Windows Trusted Root Certification Authorities Certificate Store, which **requires administrator privileges**.
+### Install the TLS certificate
```

Comment:
Installing the certificate(s) to the Windows Certificate Store is actually not needed for NATS at all. NATS will not use this Store to retrieve certificate information from.

Reply:
Do you mean we should leave out this entire  "Install the TLS certificate" section?

Reply:
Yapp, that part is totally not needed

Reply:
Doesn't the messagebroker rely on the OS (and thus the certstore) to verify certificates, or does it use a custom certificateVerificationCallback?

Reply:
Messagebroker doesn't do the validation at all for now. The plan is to have it when the TLS framework is implemented and the certificates are correctly configured. @LaurensVergote can confirm?

Reply:
Right, I wasn't sure if that was still the case